kratos/spaces/ublas_space.h

```diff
@@ -277,27 +277,34 @@ class UblasSpace
         return std::sqrt(Dot(rX, rX));
     }
 
-    static TDataType TwoNorm(MatrixType const& rA) // Frobenious norm
+    static TDataType TwoNorm(Matrix const& rA) // Frobenious norm
     {
         TDataType aux_sum = TDataType();
-#ifndef _OPENMP
```

Comment:
this should not be necessary, the `#pragma omp` clauses are usually just ignored if Kratos is compiled without OMP